tools/devops/device-tests/templates/download-artifacts.yml

```diff
@@ -14,27 +14,27 @@ steps:
   timeoutInMinutes: 1
 
 # Download the Html Report that was added by the tests job.
-- task: DownloadPipelineArtifact@2
+- task: DownloadBuildArtifacts@0
   displayName: Download html report 
   inputs:
-    patterns: '**/HtmlReport.zip'
-    allowFailedBuilds: true
-    path: $(System.DefaultWorkingDirectory)/Reports
+    allowPartiallySucceededBuilds: true
```

Comment:
Sanity check: this is the same as `allowfailedbuilds`?

Reply:
AFAIK yes